dotnetv3/dynamodb/README.md

```diff
@@ -8,33 +8,49 @@ Amazon DynamoDB is a key-value and document database that delivers single-digit
 
 ## ⚠️ Important
 
-- Running this code might result in charges to your AWS account. 
+- Running this code might result in charges to your AWS account.
 - We recommend that you grant your code least privilege, or at most the minimum
   permissions required to perform the task. For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
-- This code is not tested in all AWS Regions. For more information, see 
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+- This code is not tested in all AWS Regions. For more information, see
   [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/).
 
 ## Code examples
 
 ### Scenario
 
-  [Get started using DynamoDB tables, items, and queries](scenarios/DynamoDB_Basics/DynamoDB_Basics_Scenario/)
-
-  This application uses the basic features of DynamoDB for creating,
-  updating, and managing data. The application was created using the AWS
-  SDK for .NET version 3.7 and .NET Core 5. The application performs the
-  following tasks:
-
-  1. Creates a table with partition: year and sort:title. (`CreateTableAsync`)
-  2. Adds a single movie to the table. (`PutItemAsync`)
-  3. Adds movies to the table from moviedata.json. (`CreateBatchWrite`, `ExecuteAsync`)
-  4. Updates the rating and plot of the movie that was just added. (`UpdateItemAsync`)
-  5. Gets a movie using its key (partition + sort). (`GetItemAsync`)
-  6. Deletes a movie. (`DeleteItemAsync`)
-  7. Uses QueryAsync to return all movies released in a given year. (`QueryAsync`)
-  8. Uses ScanAsync to return all movies released within a range of years. (`ScanAsync`)
-  9. Finally, it deletes the table that was just created. (`DeleteTableAsync`)
+[Get started using DynamoDB tables, items, and queries](scenarios/DynamoDB_Basics/DynamoDB_Basics_Scenario/)
+
+This application uses the basic features of DynamoDB for creating,
+updating, and managing data. The application was created using the AWS
+SDK for .NET version 3.7 and .NET Core 5. The DynamoDB Basics scenario
+performs the following tasks:
+
+1. Creates a table with partition: year and sort:title. (`CreateTableAsync`)
+2. Adds a single movie to the table. (`PutItemAsync`)
+3. Adds movies to the table from moviedata.json. (`CreateBatchWrite`, `ExecuteAsync`)
+4. Updates the rating and plot of the movie that was just added. (`UpdateItemAsync`)
+5. Gets a movie using its key (partition + sort). (`GetItemAsync`)
+6. Deletes a movie. (`DeleteItemAsync`)
+7. Uses QueryAsync to return all movies released in a given year. (`QueryAsync`)
+8. Uses ScanAsync to return all movies released within a range of years. (`ScanAsync`)
+9. Finally, it deletes the table that was just created. (`DeleteTableAsync`)
+
+[Query a table using PartiQL](scenarios/PartiQL_Basics/DynamoDB_PartiQL_Basics_Scenario)
```

Comment:
These should be two separate scenarios, according to the [epic](https://github.com/awsdocs/aws-doc-sdk-examples/issues/2953). See https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/python/example_code/dynamodb#code-examples for example. 
the way you have it written now means that the code that surfaced for the Batch version in the guide isn't complete / or runnable.

Reply:
Split the scenario into two separate scenarios.